DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -103,11 +106,85 @@ void FillBindingData(
 
 void DmlDispatchable::Initialize()
 {
-    THROW_IF_FAILED(m_device->DML()->CompileOperator(
-        m_operator.Get(), 
-        m_desc.executionFlags, 
-        IID_PPV_ARGS(m_operatorCompiled.ReleaseAndGetAddressOf())));
-    m_operatorCompiled->SetName(std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(m_name).data());
+    if(m_desc.compileType == DmlCompileType::DmlCompileOp)
+    {
+        m_logger->LogInfo("DmlCompileOp");
+        THROW_IF_FAILED(m_device->DML()->CompileOperator(
+            m_operator.Get(), 
+            m_desc.executionFlags, 
+            IID_PPV_ARGS(m_operatorCompiled.ReleaseAndGetAddressOf())));
+        m_operatorCompiled->SetName(std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(m_name).data());
+    }
+    else
+    {
+        m_logger->LogInfo("DmlCompileGraph");
+        DML_GRAPH_DESC dmlGraphDesc = {};
+        std::vector<DML_INPUT_GRAPH_EDGE_DESC> dmlInputGraphEdges;
+        std::vector<DML_GRAPH_EDGE_DESC> dmlInputEdges;
+        
+        std::vector<DML_OUTPUT_GRAPH_EDGE_DESC> dmlOutputGraphEdges;
+        std::vector<DML_GRAPH_EDGE_DESC> dmlOutputEdges;
+        std::vector<DML_GRAPH_NODE_DESC> dmlGraphNodes;
```

Comment:
nit: just declare DML_GRAPH_NODE_DESC instead of a vector that is always size 1

Reply:
I can change it back if I need to add 2 operators at a time